modules/dns_common/dependency_injection/dependencies.rb

```diff
@@ -1,9 +1,10 @@
-require 'dns_common/dependency_injection/container'
-
 module Proxy::Dns
   module DependencyInjection
     class Dependencies
-      extend Proxy::Dns::DependencyInjection::Wiring
+      extend Proxy::DependencyInjection::Wiring
+      def self.container_instance
```

Comment:
Do you want to add a logger debug/warning message into here, so it can be removed more safely in a later release?